### DISCUSSION TITLE
Welcome to YLearn Discussions

### DISCUSSION THREAD

**Discussion by LIUZhaohan0** [DataCanvasIO/YLearn#18] [General]
Dear visitors,

Thanks for your interests in YLearn. We welcome any junior or senior developers to join our discussions. In this space, you are free to :

- Ask questions about YLearn implementation, documentation, and so on
- Share your ideas about causal related topics, eg: algorithms, applications, etc.
- Make suggestions  
- Connect with others 

We really appreciate your support, which also motivate us to keep improving YLearn project. 

With kind regards
YLearn  Team

**Comment by Elon991613** [comment]
希望能提高模型的泛化能力，所给的例子数据中具有特殊性，数值范围分布很大。另外您提供的交流方式过于闭塞，无法沟通交流，能否提供一个没有时间限制的微信群二维码，供广大读者交流。
